Update game.lua

kaborja opened the pull request:
potential fixes:
- tips now state that $5000 is required for owning a company, instead of $10
- only one digit is shown after a decimal point for the amount of money in profile
- !p command isn't extremely case-sensitive now

kaborja committed:
Update game.lua

potential fixes:
- tips now state that $5000 is required for owning a company, instead of $10
- only one digit is shown after a decimal point for the amount of money in profile
- !p command isn't extremely case-sensitive now

kaborja commented on game.lua:
```diff
@@ -379,9 +379,9 @@ function displayTips(target)
+  local p = players[upper(name)] or players[upper(name) .. "#0000"]
   if p then
-    ui.addTextArea(900, closeButton .. "<p align='center'><font size='15'><b><BV>" .. p:getName() .."</BV></b></font></p><br><b>Level:</b> " .. tostring(p:getLevel()) .. "<BL><font size='12'> [" .. tostring(p:getXP()) .. "XP / " .. tostring(calculateXP(p:getLevel() + 1)) .. "XP]</font></BL><br><b>Money:</b> $" .. formatNumber(p:getMoney()) .. "<br><br><b>Working as a</b> " .. p:getJob() , target, ...
+    ui.addTextArea(900, closeButton .. "<p align='center'><font size='15'><b><BV>" .. p:getName() .."</BV></b></font></p><br><b>Level:</b> " .. tostring(p:getLevel()) .. "<BL><font size='12'> [" .. tostring(p:getXP()) .. "XP / " .. tostring(calculateXP(p:getLevel() + 1)) .. "XP]</font></BL><br><b>Money:</b> $" .. formatNumber(float(p:getMoney())) .. "<br><br><b>Working as a</b> " .. p:getJob() ,  ...
```
I forgot to put the second argument for the float function and I'm on phone. Could you please put it as 1? Thank you.

Seniru replied:
It'd be better if you could make that change because I don't have access to the fork and I'm not sure that I'd make the change after the merge

Seniru commented:
Thanks for contributing! I'll test the code and tell my decision.

Seniru commented on game.lua:
```diff
@@ -202,7 +202,7 @@ function Player:useMed(med)
 
 function Player:updateStatsBar()
-  ui.updateTextArea(10, "<p align='right'>Money: $" .. self.money .." </p> ", self.name)
+  ui.updateTextArea(10, "<p align='right'>Money: $" .. float(self.money,1) .." </p> ", self.name)
```
You may also wrap that value with formatNumber function.

Seniru commented on game.lua:
```diff
@@ -733,7 +741,7 @@ end
 --game logic
 --creating tips
-createTip("You Need $10 To Start A New Company!", 1)
+createTip("You Need $5000 To Start A New Company!", 1)
```
Thanks!!! :D

Seniru commented on game.lua:
```diff
@@ -472,6 +472,14 @@ function formatNumber(n)
   return n
 end
 
+function float(n, digits)
```
I'm not sure what does this function do - and couldn't find a observable change in the game also. Can you tell the purpose of this, please?

kaborja replied:
Aha. It was related to this issue I had closed:
https://github.com/Seniru/TFM_Clicker/issues/25

Seniru replied:
Oh OK

Seniru replied:
You may also call this function inside formatNumber to make the thing more easy

Seniru replied:
And I reopened #25 and will close after  merging this.
(Add a commit titled closes #25 to automatically close it)

Seniru reviewed (changes requested):
Most changes are OK! I've requested some changes in comments. Please view them and make the changes.
And thanks again for contributing for this project 😄

Seniru commented:
@all-contributors please add @kaborja for code

Seniru commented:
I've reopened issues #23 #24 #25 and planned to close them after merging this PR. (You can add commits named 'closes #23, #24, #25) to close them automatically

kaborja committed:
Update game.lua

- fixed stuff involving float/formatNumber
- replaced "Atelie801" with "Atelier801"
- added a new line for the text area with "Lessons left"

Seniru commented on game.lua:
```diff
@@ -794,14 +804,14 @@ table.insert(courses, Course("Law", 35000, 80, 5, "admin"))
-table.insert(jobs, Job("Junior miner", 25, 0.1, 3, nil, "shaman", "Atelie801"))
-table.insert(jobs, Job("Cheese producer", 50, 0.15, 7, nil, "shaman", "Atelie801"))
-table.insert(jobs, Job("Cheese miner", 250, 0.2, 10, "Cheese mining", "shaman", "Atelie801"))
+table.insert(jobs, Job("Cheese collector", 10, 0.05, 1, nil, "shaman", "Atelier801"))
```
I can't think why did I typed atelie everywhere :laughing: 
Thanks for fixing

Seniru reviewed (commented):
Get ready!  This will be merged after testing

Seniru commented:
Approve

Seniru commented:
Congratulations and Thanks for the pull request 😄 🎆 🎆
